### PR TITLE
024 regform

### PR DESCRIPTION
**Pull request by RuslanTanasiichuk**

### CODE REVIEW

**Review by Darya2311 — APPROVED** [review]
Great)

**Review by Romaus — APPROVED** [review]
ok

File: src/app/regform/regform.component.ts [hunk 1]
```diff
@@ -96,15 +106,41 @@ export class RegformComponent implements OnInit {
...
     const formValueBinary = new Blob([formValueJson], {type: 'application/json'});
     const formData = new FormData();
     formData.append('form', formValueBinary);
     formData.append('idInternship', this.idInternship);
     if (this.file){ formData.append('file', this.file); }
-    this.formService.sendFormData(formData).subscribe(res => console.log(res));
+    let message: string;
+    this.formService.sendFormData(formData).subscribe(
+      data => {
+        message = 'Your application sent successfully';
```
Comment (ViktorRiabinin) [inline src/app/regform/regform.component.ts:125]: The message is not translated into the Russian language if it is selected

File: src/app/regform/accept-dialog/accept-dialog.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div>
+  <div mat-dialog-content>
+    Please make sure the data is completed correct
```
Comment (ViktorRiabinin) [inline src/app/regform/accept-dialog/accept-dialog.component.html:3]: 1. Better pass the text as a parameter to make the component re-usable
2. Translate it
3. The message sounds as if user filled the form with incorrect data.

**Pull request merged**